### PR TITLE
ci: save xdp program after build, rename helm chart

### PR DESCRIPTION
**Pull request by mbaran0v**

### CODE REVIEW

File: .deploy/helm/eupf/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: eupf
+version: 0.3.1
```
Comment (pirog-spb) [inline .deploy/helm/eupf/Chart.yaml:3]: It is a chart's or a project's version?

File: .deploy/helm/eupf/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+appVersion: autoupdate
```
Comment (pirog-spb) [inline .deploy/helm/eupf/Chart.yaml:4]: Oh. Seems like it is a project's version. Right?

File: .deploy/helm/values/dev.yaml [hunk 3]
```diff
@@ -15,23 +14,31 @@ args:
...
+
 configMaps:
   config:
     data:
       config.yml: |
         interface_name: eth0
         api_address: :8080
-        pfcp_address: :8805
-        pfcp_node_id: localhost
+        pfcp_address: :880
```
Comment (pirog-spb) [inline .deploy/helm/values/dev.yaml:28]: Check `pfcp_address` value

File: .deploy/helm/values/dev.yaml [hunk 4]
```diff
@@ -86,3 +90,56 @@ readinessProbe:
     port: http
   initialDelaySeconds: 5
   periodSeconds: 10
+
+# extraDeploy:
```
Comment (pirog-spb) [inline .deploy/helm/values/dev.yaml:94]: Why we need this config commented here?

Comment (mbaran0v) [reply .deploy/helm/values/dev.yaml:94]: because work in progress

File: .deploy/helm/values/dev.yaml (hunk 3 above)
Comment (mbaran0v) [reply .deploy/helm/values/dev.yaml:28]: ok

File: .deploy/helm/eupf/Chart.yaml (hunk 1 above)
Comment (mbaran0v) [reply .deploy/helm/eupf/Chart.yaml:3]: chart version

File: .deploy/helm/eupf/Chart.yaml (hunk 2 above)
Comment (mbaran0v) [reply .deploy/helm/eupf/Chart.yaml:4]: app version (docker image tag)

**Review by pirog-spb — APPROVED** [review]

**Pull request merged**